File: algo-c-to-_/examples/change.lua
--
--	from src/change.c
--
--	int change(int, int)		to	changeR
--	int change1(int, int)		to	changeL
--

local M = require 'change'

local changeR, changeL = M.changeR, M.changeL

for i=0,500,5 do
	local a, b = changeR(i, i), changeL(i)

	assert(a == b, ("changeR(i,i) ~= changeL(i) (i:%d, changeR:%d, changeL:%d)"):format(i, a, b))

	print(("%4d %8d"):format(i, a))
end

File: algo-c-to-_/examples/change_.py
#
#	from src/change.c
#
#	int change(int, int)		to	changeR
#	int change1(int, int)		to	changeL
#

from change import changeR, changeL

for i in range(0, 501, 5):
    a, b = changeR(i, i), changeL(i)

    assert a == b, "changeR(i,i) != changeL(i) (i:{0}, changeR:{1}, changeL:{2})".format(i, a, b)

    print("{0:4d} {1:8d}".format(i, a))
